src/com/vinnik/kate/MultithreadingSorter.java

```diff
@@ -0,0 +1,63 @@
+package com.vinnik.kate;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+/** Class, which creates given number of threads, and measures time they need to sort random array. */
+public class MultithreadingSorter {
+    private static final int ARRAY_SIZE = 500;
+    private static final int UPPER_BOUND_RANDOM = 10000;
+    private static final int NANOSECONDS_IN_MILLISECONDS = 1000000;
+
+    /** Creates given number of threads and executes them. */
+    public static void measureTimeOfSorting(final int amountOfThreads) {
+        for (int i = 0; i < amountOfThreads; i++) {
+            final ThreadSort thread = new ThreadSort(i);
+            thread.start();
+        }
+    }
+
+
+    final protected static class ThreadSort extends Thread {
+        private final int[] numbersToSort;
```

Comment:
Let's focus on the code's design now. I don't think sorting amount of numbers to sort within the class variable is sufficient in this case. Wouldn't it be better to have a method that accepts amount of numbers to sort?